stwcs/updatewcs/utils.py

```diff
@@ -1,10 +1,17 @@
 import os
+import string
 from stsci.tools import fileutil
 
 import logging
 logger = logging.getLogger("stwcs.updatewcs.utils")
 
 
+# These are keywords that may be present in headers created by WCSLIB.
+# However, HST keeps them (or similar information) in the primary header and so
+# they should not be managed by STWCS when copying/archiving an HST WCS:
+EXCLUDE_WCSLIB_KWDS = ['EQUINOX', 'LONPOLE', 'LATPOLE', 'RESTWAV', 'RESTFRQ']
```

Comment:
`updatewcs` imports code from `wcsutil` but `wcsutil` does not import from `updatewcs`. To keep things separate should this and the function below be moved to `wcsutil`?

Reply:
I do not see why. I think it would be a great to have them independent (as much as possible). Where would you suggest in `wcsutils`?